Gemfile

```diff
@@ -10,12 +10,13 @@ gem 'dotenv-rails'
 gem 'enum_help'
 gem 'figaro'
 gem 'foreman', require: false
-gem 'hashie'
+gem 'hashie', '~> 3.5', '>= 3.5.5'
```

Comment:
I think this can just be `gem 'hashie', '>= 3.5.5'` -- I am not sure what bundler does with two args, usually just see 1